packages/boba/contracts/contracts/bridges/L1NFTBridge.sol

```diff
@@ -72,7 +72,7 @@ contract L1NFTBridge is iL1NFTBridge, CrossDomainEnabled, ERC721Holder, Reentran
      **********************/
 
     modifier onlyOwner() {
-        require(msg.sender == owner || owner == address(0), 'Caller is not the owner');
+        require(msg.sender == owner, 'Caller is not the owner');
```

Comment:
this was modified after discussing it with @souradeep-das.
the `onlyOwner` modifier was not used correctly or functions could be used when the bridge was yet initialized.

Reply:
the gap between deployment -> init is small, but as the wise say (@souradeep-das)
```
the proxy deploy script - deploys and next step is initialize, so as long as script runs and no one knows contract addresses its fine, but still should not be grouped and categorized as initialization step bias, needs to be changed 
```

Reply:
😝